<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1024</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>BINS</string>
  </property>
  <property name="iconSize">
   <size>
    <width>32</width>
    <height>32</height>
   </size>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="CentralTab">
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="toolTip">
       <string/>
      </property>
      <property name="toolTipDuration">
       <number>-1</number>
      </property>
      <property name="styleSheet">
       <string notr="true">QTabWidget::pane { 
 min-width: 10ex;
 padding: 5px;
 border: 2px solid #C4C4C7;
border-bottom-color:rgb(75, 177, 255);
	border-top-color: rgb(75, 177, 255);
	border-left-color: rgb(75, 177, 255);
	border-right-color: rgb(75, 177, 255);
} 
QTabBar::tab {
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                stop: 0 #E1E1E1, stop: 0.4 #DDDDDD,
                                stop: 0.5 #D8D8D8, stop: 1.0 #D3D3D3);
    border: 2px solid #C4C4C7;
	border-bottom-color: rgb(255, 143, 14);
	border-top-color: rgb(255, 143, 14);
	border-left-color: rgb(255, 143, 14);
	border-right-color: rgb(255, 143, 14);
    border-top-left-radius: 2px;
    border-top-right-radius: 2px;
    min-width: 10ex;
    padding: 5px;
}

QTabBar::tab:selected, QTabBar::tab:hover {
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                stop: 0 #fafafa, stop: 0.4 #f4f4f4,
                                stop: 0.5 #e7e7e7, stop: 1.0 #fafafa);
}

QTabBar::tab:selected {
    border-color: #9B9B9B;
    border-bottom-color: #C2C7CB; 
   border: 2px solid #C4C4C7;
	border-bottom-color:rgb(79, 255, 15);
	border-top-color: rgb(79, 255, 15);
	border-left-color: rgb(79, 255, 15);
	border-right-color: rgb(79, 255, 15);
    border-top-left-radius: 2px;
    border-top-right-radius: 2px;
    min-width: 10ex;
    padding: 5px;
}
</string>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="Connection">
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>9</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <attribute name="title">
        <string>Подключение</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QGroupBox" name="COMBox">
          <property name="font">
           <font>
            <family>Segoe UI</family>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="title">
           <string>Текущие настройки подключения</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="0">
            <layout class="QGridLayout" name="settings_grid">
             <item row="7" column="0" colspan="2">
              <widget class="QPushButton" name="open_port_button">
               <property name="font">
                <font>
                 <family>Segoe UI</family>
                 <pointsize>11</pointsize>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">*{
    border-width: 1px;
    border-style: solid;
    border-radius: 4px;
}

*:closed{
	border-color:rgb(0, 0, 0);
	background:rgb(157, 255, 157);
}

*:pressed{
	border-color:rgbrgb(0, 139, 46);
	background:rgbrgb(0, 139, 46);
}</string>
               </property>
               <property name="text">
                <string>Открыть</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="access_com">
               <property name="font">
                <font>
                 <family>Segoe UI</family>
                 <pointsize>9</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Список доступных устройств</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item row="6" column="0">
              <spacer name="verticalSpacer">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
             <item row="7" column="2">
              <widget class="QPushButton" name="close_port_button">
               <property name="font">
                <font>
                 <family>Segoe UI</family>
                 <pointsize>11</pointsize>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">*{
    border-width: 1px;
    border-style: solid;
    border-radius: 4px;
}

*:closed{
	border-color:rgb(0, 0, 0);
	background:rgb(255, 166, 131)
}
*:pressed{
	border-color:rgb(255, 179, 180);
	background:rgb(128, 0, 0);
}</string>
               </property>
               <property name="text">
                <string>Закрыть</string>
               </property>
              </widget>
             </item>
             <item row="5" column="2">
              <widget class="QComboBox" name="com_box">
               <property name="minimumSize">
                <size>
                 <width>101</width>
                 <height>21</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <family>Segoe UI</family>
                 <pointsize>9</pointsize>
                </font>
               </property>
              </widget>
             </item>
             <item row="2" column="0" colspan="3">
              <widget class="QTextBrowser" name="com_text">
               <property name="minimumSize">
                <size>
                 <width>331</width>
                 <height>192</height>
                </size>
               </property>
              </widget>
             </item>
             <item row="8" column="0" colspan="3">
              <widget class="QLineEdit" name="console_com">
               <property name="font">
                <font>
                 <family>Segoe UI</family>
                 <pointsize>10</pointsize>
                </font>
               </property>
              </widget>
             </item>
             <item row="5" column="0" colspan="2">
              <widget class="QLabel" name="cur_com">
               <property name="font">
                <font>
                 <family>Segoe UI</family>
                 <pointsize>9</pointsize>
                </font>
               </property>
               <property name="layoutDirection">
                <enum>Qt::LeftToRight</enum>
               </property>
               <property name="autoFillBackground">
                <bool>false</bool>
               </property>
               <property name="text">
                <string>Текущее выбранное устройство</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item row="1" column="2">
              <widget class="QPushButton" name="refresh_port_button">
               <property name="text">
                <string>Обновить</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QGroupBox" name="groupBox">
          <property name="minimumSize">
           <size>
            <width>381</width>
            <height>511</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Segoe UI</family>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="title">
           <string>Отображение данных</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_4">
           <item row="0" column="0">
            <layout class="QGridLayout" name="gridData">
             <item row="0" column="0">
              <widget class="QTableView" name="table_proto">
               <attribute name="horizontalHeaderDefaultSectionSize">
                <number>200</number>
               </attribute>
               <attribute name="horizontalHeaderMinimumSectionSize">
                <number>36</number>
               </attribute>
               <attribute name="verticalHeaderDefaultSectionSize">
                <number>30</number>
               </attribute>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="SeeWindow">
       <attribute name="title">
        <string>Параметры ориентации</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_7">
        <item row="0" column="1">
         <widget class="QGroupBox" name="groupBox_2">
          <property name="minimumSize">
           <size>
            <width>700</width>
            <height>511</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Segoe UI</family>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="title">
           <string>Визуализация</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_9">
           <item row="0" column="0">
            <layout class="QGridLayout" name="gridLayout_6">
             <item row="0" column="0">
              <widget class="GLWidget" name="model" native="true">
               <property name="focusPolicy">
                <enum>Qt::ClickFocus</enum>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QGroupBox" name="COMBox_2">
          <property name="font">
           <font>
            <family>Segoe UI</family>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="title">
           <string>Данные инерциальной навигационной системы</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_8">
           <item row="0" column="0">
            <layout class="QGridLayout" name="gridLayout_5">
             <item row="2" column="0">
              <widget class="QGroupBox" name="COMBox_4">
               <property name="font">
                <font>
                 <family>Segoe UI</family>
                 <pointsize>9</pointsize>
                </font>
               </property>
               <property name="title">
                <string>Компас</string>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QGroupBox" name="COMBox_3">
               <property name="font">
                <font>
                 <family>Segoe UI</family>
                 <pointsize>9</pointsize>
                </font>
               </property>
               <property name="title">
                <string>Акселерометр</string>
               </property>
               <widget class="QWidget" name="gridLayoutWidget_3">
                <property name="geometry">
                 <rect>
                  <x>10</x>
                  <y>20</y>
                  <width>241</width>
                  <height>61</height>
                 </rect>
                </property>
                <layout class="QGridLayout" name="gridLayout_10">
                 <item row="0" column="0">
                  <widget class="QLabel" name="access_com_2">
                   <property name="font">
                    <font>
                     <family>Segoe UI</family>
                     <pointsize>9</pointsize>
                    </font>
                   </property>
                   <property name="text">
                    <string>Тангаж</string>
                   </property>
                   <property name="alignment">
                    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="0">
                  <widget class="QLabel" name="access_com_3">
                   <property name="font">
                    <font>
                     <family>Segoe UI</family>
                     <pointsize>9</pointsize>
                    </font>
                   </property>
                   <property name="text">
                    <string>Крен</string>
                   </property>
                   <property name="alignment">
                    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="1">
                  <widget class="QDoubleSpinBox" name="pitch_sb">
                   <property name="prefix">
                    <string/>
                   </property>
                   <property name="suffix">
                    <string>°</string>
                   </property>
                   <property name="minimum">
                    <double>-999.000000000000000</double>
                   </property>
                   <property name="maximum">
                    <double>999.000000000000000</double>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="1">
                  <widget class="QDoubleSpinBox" name="pitch_sb_2">
                   <property name="prefix">
                    <string/>
                   </property>
                   <property name="suffix">
                    <string>°</string>
                   </property>
                   <property name="minimum">
                    <double>-999.000000000000000</double>
                   </property>
                   <property name="maximum">
                    <double>999.000000000000000</double>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QGroupBox" name="COMBox_5">
               <property name="font">
                <font>
                 <family>Segoe UI</family>
                 <pointsize>9</pointsize>
                </font>
               </property>
               <property name="title">
                <string>Гироскоп</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Работа с ГНСС</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Разработка</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <action name="action">
   <property name="text">
    <string>Информация о программе</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>GLWidget</class>
   <extends>QWidget</extends>
   <header>glwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
